Changelog.md

```diff
@@ -1,3 +1,13 @@
+### 1.3.0 / 2019-03-01
```

Comment:
Technically this is "Development"

Reply:
should, I remove this diff too per your version bump on release comment?

Reply:
Remove the version reference, change log itself is fine.

Reply:
cool, done